Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -200,17 +200,35 @@ def remediation_command_list_request(self, time_range: Dict[str, Any], alert_ids
     def alert_remediate_request(self, alert_id: str):
         self._http_request('PATCH', f'alert/remediation/{alert_id}', resp_type='response')
 
+    def remove_additional_resource_fields(self, input_dict, keys):
+        if isinstance(input_dict, dict):
+            return {k: self.remove_additional_resource_fields(v, keys) for k, v in input_dict.items() if k not in keys}
+        elif isinstance(input_dict, list):
+            return [self.remove_additional_resource_fields(element, keys) for element in input_dict]
+        else:
+            return input_dict
+
     def config_search_request(self, time_range: Dict[str, Any], query: str, limit: Optional[int] = None,
                               search_id: Optional[str] = None, sort_direction: Optional[str] = None,
-                              sort_field: Optional[str] = None):
+                              sort_field: Optional[str] = None, include_resource_json: Optional[str] = 'true',
+                              include_additional_resource_fields: Optional[str] = 'true',
```

Comment:
If handling `remove_additional_resource_fields` is not moved out from this function, should be changed
```suggestion
                              include_additional_resource_fields: Optional[bool] = True,
```
otherwise `if not 'false' == False`

Reply:
[Moved](https://github.com/demisto/content/pull/32606/commits/5fa21493d916ad6da2ccfe22910449fdbc49d6d5).